### DISCUSSION TITLE
Java Client for Qdrant

### DISCUSSION THREAD

**Discussion by zhenlohuang** [qdrant/qdrant#1178] [Ideas]
**Is your feature request related to a problem? Please describe.**
We are building a solution based on Java, so that it would be better if we can have java client.

**Describe the solution you'd like**
It would better if you can provide Java client offically. 

**Describe alternatives you've considered**
we use openapi/swagger spec to generate Java client.

**Additional context**
no.

**Comment by generall** [comment]
Another alternative is to generate Java client using gRPC definitions, it is usually more reliable.

**Comment by Jotschi** [comment]
I wrote a gRPC / REST client for java - maybe it fits your needs.

https://github.com/metaloom/qdrant-java-client
